### PR TITLE
DOC-1757: Updates so the user would understand how to update the default options

### PR DESCRIPTION
**Pull request by kemister85**
Related Ticket: [DOC-1757](https://ephocks.atlassian.net/browse/DOC-1757?atlOrigin=eyJpIjoiY2FiYWU2OTBjYmZkNGU2ZGI5YTMyZTk0ZjVkZTRiYzEiLCJwIjoiaiJ9)

Description of Changes:
* Added documentation guild for user to user overrideDefault with customOptions for version 6 cloud.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] `modules/ROOT/nav.adoc` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [ ] Documentation Team Lead has reviewed

### CODE REVIEW

**Review by lnewson — CHANGES_REQUESTED** [review]
This isn't how we need to update API docs sorry @kemister85. API docs are autogenerated, then committed here and need to be updated in the TinyMCE source code instead.

**Comment by lnewson** [comment]
See https://github.com/tinymce/tinymce/blob/9dc5a95e81e8e424818b5adc29a2fde8c5f0fa9c/modules/tinymce/src/core/main/ts/api/EditorManager.ts#L272-L277 as to where this needs to be changed instead. It also uses moxiedoc syntax instead (which is fairly close to jsdoc).

File: modules/ROOT/pages/apis/tinymce.root.adoc [hunk 1]
```diff
@@ -421,7 +421,27 @@ one array list into another.
 ----
 overrideDefaults(defaultOptions: Object)
 ----
-Overrides the default options for editor instances.
+`overrideDefaults` will replace any of the defaults set by a previous call to the `overrideDefaults` function. 
+When using the cloud some of the defaults are required for the cloud based editor to function.
+
+Therefore when using `overrideDefaults` with the cloud based editor, the user will need to combine the previous defaults `tinymce.defaultOptions` with the new options that you want to integrate.
```
Comment (brian-forte) [inline modules/ROOT/pages/apis/tinymce.root.adoc:427]:
```suggestion
`overrideDefaults` replaces any of the defaults set by a previous call to the `overrideDefaults` function.

When using the cloud, some of these defaults are required for the cloud-based editor to function.

Therefore, when using `overrideDefaults` with the cloud-based editor, combine the previous defaults, `tinymce.defaultOptions`, with the new options that you want to integrate.
```
Suggested copy-edits.

Comment (brian-forte) [reply modules/ROOT/pages/apis/tinymce.root.adoc:427]: Also, is _cloud_ above generic, or is it a reference to the Tiny Cloud?

If the latter, `s/cloud/{cloudname}/` should be run over the new text.

**Comment by kemister85** [comment]
Note: PR to be closed >> assigned to @MitchCTiny to be reopened [DOC-1757](https://ephocks.atlassian.net/browse/DOC-1757?atlOrigin=eyJpIjoiNWY3NzNlZjgwZDMxNGU0MTlhMzg0NGE3ZGZlNTRmNTAiLCJwIjoiaiJ9)

**Pull request closed without merging**